src/libs/residfp/SID.cpp

```diff
@@ -114,14 +114,15 @@ void SID::voiceSync(bool sync)
 
     for (int i = 0; i < 3; i++)
     {
-        const unsigned int freq = voice[i]->wave()->readFreq();
+        WaveformGenerator* const wave = voice[i]->wave();
```

Comment:
do we prefer `auto` here?

Reply:
In general we want to leave upstream code as is.  That said, I _have_ applied some warning fixes, but that's only because the upstream developer is resistant to making those changes.